modules/ossm-config-control-plane-infrastructure-node.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-control-plane-infrastructure-node_{context}"]
+== Configuring all {SMProductShortName} Control Plane components to run on infrastructure nodes
+
+This task should only be performed if all of the components deployed by the {SMProductShortName} control plane (including Istiod, Ingress Gateway, and Egress Gateways) along with optional elements (such as Prometheus, Grafana, and Distributed Tracing) will run on infrastructure nodes. 
+
+If the control plane will run on a worker node, skip this task.
+
+.Procedure
+
+. Edit the `ServiceMeshControlPlane` resource to present the YAML file in the editor. In this example, `<name>` represents the name of the `ServiceMeshControlPlane` resource.
++
+[source,terminal]
+----
+$ oc -n istio-system edit smcp <name>
+----
+
+. To run all of the Service Mesh components deployed by the SMCP on infrastructure nodes add `nodeSelector` and `tolerations` to `spec.runtime.defaults.pod` in the `ServiceMeshControlPlane` resource.
```

Comment:
```suggestion
. To run all of the {SMProductShortName} components deployed by the {SMProductShortName} control plane (SMCP) on infrastructure nodes, add the `nodeSelector` and `tolerations` fields to the `runtime.defaults.pod` spec in the `ServiceMeshControlPlane` resource:
```

suggestions, recommend always using the {SMProductShortName} attribute where possible, and if this is the first instance of SMCP it should be defined

Reply:
Done